projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -1031,6 +1031,11 @@ For examples of how to define and use new extension points, please see the code
 || Type | Extension Point | Description |
 | ``Decider`` | ``decide_executeGesture`` | Notifies when a gesture is about to be executed, allowing other code to decide if it should be. |
 
+++ logHandler ++[logHandlerExtPts]
+
+|| Type | Extension Point | Description |
+| ``Action`` | ``_onErrorSoundRequested`` | Triggered every time an error sound needs to be played. This extension point should not be used directly but retrieved calling ``getOnErrorSoundRequested()`` instead. |
```

Comment:
I'd suggest to avoid mentioning `_onErrorSoundRequested` explicitly both here and in the change log. The fact that this exists as a private variable is irrelevant to developers, and they should only access it via the getter function. By mentioning it were essentially committing to the private part of the API.

Reply:
I disagree - the dev guide and changes for developers is for core developers and other contributors, not just add-on developers. The distinction here on usage and API boundary is fairly clear.